invenio_cli/helpers/docker_helper.py

```diff
@@ -105,9 +108,29 @@ def destroy_containers(self):
         # Close logging pipe
         logpipe.close()
 
-    def copy(self, src_file, dst_path, container):
+    def _normalize_name(self, app_name):
+        """Normalize the container name according to the compose version.
+
+        Docker-Compose introduced support for dash and underscore in
+        version 1.21.0.
+        """
+        dc_version_command = subprocess.Popen(['docker-compose', '--version'],
+                                              stdout=subprocess.PIPE)
+
+        dc_version_string = dc_version_command.communicate()[0]
+
+        groups = re.search(r'1.[0-9]*.[0-9]*', dc_version_string)
+        dc_version = groups.group(0)
+
+        if dc_version < DOCKER_COMPOSE_VERSION_DASH:
+            return re.sub(r'[^-_a-z0-9]', '', app_name)
+        else:
+            return app_name
+
+    def copy(self, src_file, dst_path, app_name):
         """Copy a file into the path of the specified container."""
-        container = self.docker_client.containers.get(container)
+        container_name = '{}_web-ui_1'.format(self._normalize_name(app_name))
+        container = self.docker_client.containers.get(container_name)
```

Comment:
Idle thought: was there a way to get container by ID (that we could *actually* use) to get around needing this?

There are minuscule things worth a comment at least:
- `dc_version < DOCKER_COMPOSE_VERSION_DASH` will fail once  `dc_version = '1.1XX.X'` - which won't realistically happen, but is worth putting a comment to assuage other developers that may be concerned by low-probability edge cases
- `web` introduces a dependency on the docker-compose.yml file. Which again is fine, as long as we keep that in mind and don't change the names in the docker-compose.yml file.